docs/layers.md

```diff
@@ -839,18 +837,18 @@ To improve performance, some road segments are merged at low and mid-zooms. To f
 * `ferry`: See kind list below.
 * `highway`: See kind list below.
 * `is_bicycle_related`: Present and `true` when road features is a dedicated cycleway, part of an OSM bicycle network route relation, or includes cycleway infrastructure like bike lanes or designed for shared use.
-* `is_bridge`: `yes` or `no`
+* `is_bridge`: `true`, default is `false` if not present.
```

Comment:
For `is_bridge`, `is_link` and `is_tunnel` they should never be exported if they're `false`. I've updated the docs to try and make that clearer in #825.